readme.md

```diff
@@ -28,13 +28,18 @@ $ cargo run --example read_song -- examples/songs/DEFAULT.m8s
 ## TODO
 - Add song groove, scale, note_preview
 - Add settings: output/speaker volume
-- Support writes?
 - Throw more parse errors
-- Interpret FXCommand based on Instrument
 - Displays: MixerSettings, EffectsSettings, Instrument, MidiSettings, MidiMapping
 
 ## Changelog
 
+### 0.3.1
+
+ - Fixing visibility of Eq types (was private in 0.3)
+ - FmAlgo content is now publicly visible
+ - Each instrument filter type can now be accessed through a methdo.
```

Comment:
Typo: `method`. Being explicit in changelogs is good: "Each instrument filter type can now be accessed through the `filter_types` method" is preferable.